index.html

```diff
@@ -10,25 +11,24 @@
 
       gtag('config', 'G-H5914B49ZF');
     </script>
-    <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Portfolio</title>
```

Comment:
- [ ] Please re-write the title tag as the requirement say  see the screenshot 👇

![kkk](https://user-images.githubusercontent.com/61562626/130761856-609ce6b7-353a-4fc3-827f-9ca3cbf5424d.jpeg)